src/main/java/com/anaplan/client/AnaplanService.java

```diff
@@ -0,0 +1,228 @@
+package com.anaplan.client;
+
+import com.anaplan.client.ex.ActionsNotFoundException;
+import com.anaplan.client.ex.ProcessesNotFoundException;
+import com.anaplan.client.ex.ServerFilesNotFoundException;
+import com.google.common.base.Strings;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.Serializable;
+import java.net.URI;
+import java.net.URISyntaxException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Wrapper class of the Anaplan service providing the interfaces for a client to connect with
+ * Anaplan server.
+ */
+public class AnaplanService implements Serializable {
```

Comment:
does this need to be Serializable? It seems more like a utility class with only static methods

Reply:
No, it's cleaned up.